src/pages/industry.astro

```diff
@@ -0,0 +1,155 @@
+---
+import IndustryLayout from "../layouts/IndustryLayout.astro";
+
+import Speaker from "../components/people/SpeakerSmall.astro";
+
+import Sponsor from "../components/sponsor/sponsor.astro";
+import SponsorTier from "../components/sponsor/sponsor-tier.astro";
+
+import TimetableItem from "../components/timetable/timetable-item.astro";
+import Timetable from "../components/timetable/timetable.astro";
+
+import Markdown from "../components/Markdown.astro";
+import Button from "../components/Button.astro";
+import ButtonGroup from "../components/ButtonGroup.astro";
+---
+
+<IndustryLayout>
+    <Markdown slot="about-industry">
+        <h2 class="header space-y-6" id="about">About the industry track</h2>
+        <p>The industry track is back at RustWeek 2025!</p>
+        <p>
+            Following the succes of the first edition, this track is dedicated to decision makers exploring the adoption
+            of Rust in their organisation. 
+        </p>
+        <p>
+            Date & time: <strong>Tuesday May 13, 9:30 - 16:30 CET</strong>.
+        </p>
+        <h3 class="venue">City & venue</h3>
+        <p class="venue">
+            Kinepolis Jaarbeurs Utrecht, <br /> Jaarbeursboulevard 300, <br
+            /> 3521 BC Utrecht, The Netherlands
+        </p>
+    </Markdown>
+    <Timetable slot="timetable-day">
+        <TimetableItem
+            time="9.30"
+            title="Registration"
+        />
+        <TimetableItem
+            time="10.00"
+            title="Opening"
+            speaker="Hugo van de Pol"
+        />
+        <TimetableItem
```

Comment:
inconsistent indentation?

Reply:
Fixed by hugo in https://github.com/rustnl/rustnl2025/pull/25/commits/66e5504b2d77e92cb4147633dc033a431215036e